ndk-context/src/lib.rs

```diff
@@ -86,3 +86,15 @@ pub unsafe fn initialize_android_context(java_vm: *mut c_void, context_jobject:
     });
     assert!(previous.is_none());
 }
+
+/// Removes the [`AndroidContext`]. It is destroyed by [__ndk-glue__](https://crates.io/crates/ndk-glue)
+/// when the activity is finished and destroyed.
+///
+/// # Safety
+///
+/// This function must only be called after [`initialize_android_context()`] when the activity
+/// is subsequently destroyed.
+pub unsafe fn reset_android_context() {
```

Comment:
how about `destroy_android_context`? I think it better mirrors the naming used by android and sounds more like the inverse of initialize to me than reset.

Reply:
That was my first thought, but this function doesn't really "destroy" anything as much as it "gives up" some pointer references to things. It is a better match for `on_destroy` indeed. Perhaps `release` neatly complements `initialize`?

Reply:
dropping dangling pointers seems to be destroying something, but ok :man_shrugging:

Reply:
We're "destroying" the 8 bytes of pointer data, but not the pointee :P